### ISSUE TITLE
Issue when using with cypress-har-generator

### ISSUE THREAD

**Issue by MohamadKh75** [elaichenkov/cypress-plugin-init#24]
Hey

Thanks for your awesome project!

I'm going to use it with [`cypress-har-generator`](https://github.com/NeuraLegion/cypress-har-generator) and I'm getting an error:

```js
import pkg from '@neuralegion/cypress-har-generator';

initPlugins(on, [mochaReporter, pkg.install], config)
```

and I get:

```
InvalidEventHandlerError: The handler for the event `task` must be an object
```

**Comment by alyssa-glean** [comment]
I'm getting the same error about the 'task' handler. In our case, it's our own `on('task')` that's blowing it up. It's the right syntax so not sure why it's getting sad:

```
on('task', {
    'match-screenshot': async (args) => {
        ...stuff...
    },
    'gmail:get-message': async (args) => {
        ...stuff...
    }
})
```

**Comment by elaichenkov** [comment]
Thank you for reporting the issue with examples. I'll take a look at it and let you know.

**Comment by elaichenkov** [comment]
Hey @MohamadKh75 and @alyssa-glean
I've just fixed the issue and published the fix in the `0.0.8` version. So, please update the library and let me know if you need anything else.
